projects/gsc/experiments/base.py

```diff
@@ -68,7 +68,7 @@
     # Training batch size
     batch_size=BATCH_SIZE,
     # Validation batch size
-    val_batch_size=BATCH_SIZE,
+    val_batch_size=sys.maxsize,
```

Comment:
You should test run this. I've seen an error before when initializing a dataloader with `batch_size=sys.maxsize`. Could just be easier to set it to 1000 or something like that.

Reply:
Thanks - fixed.